front/app/api/ideas/useDeleteIdea.ts

```diff
@@ -34,8 +35,9 @@ const useDeleteIdea = () => {
       queryClient.invalidateQueries({
         queryKey: ideaImagesKeys.item({ ideaId }),
       });
+      queryClient.invalidateQueries({ queryKey: projectsKeys.all() });
```

Comment:
Do we really need to invalidate the entire projects cache here? Shouldn't it be enough to invalidate the project the idea is associated with? Or at least all the `items` perhaps? Or does it also affect the list endpoint?

Reply:
Ah I see, I think in that case I think you are right - we are displaying the idea count from the list so we need to invalidate all lists probably. And it would be nice to invalidate just the project with the particular id but in that case, we don't have it. One way to be to pass it to the hook and take it from there, but in this case, I'm not sure if it's not more trouble than it's worth 🤔